docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md

```diff
@@ -14,6 +14,25 @@ title: Test backing image
 9. Create and attach a volume with the backing image. Verify the data content is correct.
 10. Do cleanup.
 
+### Upload via Rancher URI
+<sup>Related issue: [3129](https://github.com/longhorn/longhorn/issues/3129) with fix in Longhorn v1.3.0+</sup>
+1. With Rancher v2.6.x create a cluster and install Longhorn
+   - or import an existing cluster with Longhorn installed
+2. Go into the cluster
+3. Click Longhorn from the Rancher menu, the sidebar on the left
```

Comment:
I think you mean Longhorn in the cluster explorer mode. It's better to explicitly claim that. Similar for the ingress.

Reply:
I've updated the document to use the official wording: _Cluster Explorer_ and _Apps & Marketplace_ from Rancher v2.6.x